Google.Api.Generator/Generation/MethodDetails.cs

```diff
@@ -297,20 +296,18 @@ public sealed class FieldExtraction
             }
 
             public static RoutingHeader CreateOneFieldNoRegex(string encodedName, IEnumerable<FieldDescriptor> fields) =>
-                new()
+                new RoutingHeader()
```

Comment:
Nit (for me to fix in a follow-up PR) - we normally omit the `()` when it's not needed. (Here and below for FieldExtraction.)

Reply:
:+1: